### PR TITLE
ab/xlsx file reading

### PR DESCRIPTION
**Pull request by Sandman148**
Added reading user data from xlsx files.

### CODE REVIEW

File: src/test/java/com/google/testdata/TestUser.java [hunk 1]
```diff
@@ -1,29 +1,92 @@
...
 
-    public static String getLastName() {
-        return lastName;
+    /**
+     * Gets user data from .xslx file.
+     * Returns data for single user from 'userNumber' row
+     */
+    private List<String> getUserData(int userNumber) {
+        try {
+            File userFile = new File("D://googletest/src/test/resources/users.xlsx");
```
Comment (AndrewPonyk) [inline src/test/java/com/google/testdata/TestUser.java:48]: Should be as relative path

File: src/test/java/com/google/testdata/TestUser.java [hunk 2]
```diff
@@ -1,29 +1,92 @@
...
+                    Cell cell = cellIterator.next();
+                    userData.add(cell.getStringCellValue());
+                }
+
+                usersData.add(userData);
+                userData = new ArrayList<String>();
+
+            }
+
+            return usersData.get(userNumber);
```
Comment (AndrewPonyk) [inline src/test/java/com/google/testdata/TestUser.java:83]: Maybe skip unnecessary user accounts.

File: src/test/java/com/google/testdata/TestUser.java (hunk 1 above)
Comment (Sandman148) [inline src/test/java/com/google/testdata/TestUser.java:48]: Will do that in scope of separate pull request.

File: src/test/java/com/google/testdata/TestUser.java (hunk 2 above)
Comment (Sandman148) [inline src/test/java/com/google/testdata/TestUser.java:83]: Will do that in scope of separate pull request.
